app/assets/stylesheets/_typography.css.scss

```diff
@@ -0,0 +1,26 @@
+
+
+body p, h1, h2, h3, h4, h5, h6, .p, .h1, .h2, .h3, .h4, .h5, .h6 {
+  font-family: 'Titillium Web', sans-serif;
+  font-weight: 500;
+  line-height: 2;
+  color: $muteBlack;
+}
```

Comment:
Rule declaration should be followed by an empty line<br>Each selector in a comma sequence should be on its own line